README.md

```diff
@@ -134,6 +134,96 @@ class ArtistFilter(AutocompleteFilter):
         return reverse('admin:custom_search')
 ```
 
+Shortcut for creating filters:
+------------------------------
+
+It is now possible to use the `ACFilter` shortcut to create filters
+on the fly, as shown below. Nested relations are supported, with
+no need to specify the model.
+
+```python
+from django.contrib import admin
+from admin_auto_filters.filters import ACFilter
+
+
+class AlbumAdmin(admin.ModelAdmin):
+    list_filter = [
+        ACFilter('Artist', 'artist', 'admin:custom_search', True)
+    ]
+
+    def get_urls(self):
+        """As above..."""
+```
+
+Customizing Widget Text
+-----------------------
+
+It is also possible to customize the text displayed in the filter
+widget, to use something other than `str(obj)`. This needs to be
+configured for both the dropdown endpoint and the widget itself.
+
+In your views.py, override display_text:
+
+```python
+from admin_auto_filters.views import AutocompleteJsonView
+
+
+class CustomSearchView(AutocompleteJsonView):
+
+    @staticmethod
+    def display_text(obj):
+        return obj.my_str_method()
+
+    def get_queryset(self):
+        """As above..."""
+```
+
+In an AutocompleteFilter in your models.py, specify the form_field:
```

Comment:
IMO filters should not be in models module, rather in admin :slightly_smiling_face: 
(They are not doing any database querying or data modelling, but implement something for django-admin)

Reply:
Completely agree - that's a typo on my part. Definitely should be admin.py! :grinning: